02-household-queries/run.py

```diff
@@ -0,0 +1,65 @@
+import dotenv
+from langchain_community.embeddings import SentenceTransformerEmbeddings
+from langchain_community.llms import GPT4All
+from langchain_community.vectorstores import Chroma
+from langchain_google_genai import ChatGoogleGenerativeAI, GoogleGenerativeAIEmbeddings
+import os
+
+from ingest import ingest_call
+from retrieval import retrieval_call
+
+dotenv.load_dotenv()
+
+print("""
+Which embedding function would you like to use?
+1. Google Gen AI (default)
+2. all-MiniLM-L6-v2
+      """)
+embedding_choice = input()
+# Set embedding function
+if embedding_choice == 2 or embedding_choice == "all-MiniLM-L6-v2":
+    embeddings = SentenceTransformerEmbeddings(model_name="all-MiniLM-L6-v2")
+else:
+    embeddings = GoogleGenerativeAIEmbeddings(model="models/embedding-001")
+
+print("""
+Which LLM would you like to use? 
+1. Gemini (default)
+2. Mistral
```

Comment:
```suggestion
2. Mistral (via GPT4All)
```
I'll be adding another option for `Mistral (Ollama)`